user-interface/src/lib/components/uswds/DateRangePicker.test.tsx

```diff
@@ -93,4 +93,50 @@ describe('Test DateRangePicker component', async () => {
     expect(startDateText).toHaveAttribute('max', '01/01/2035');
     expect(endDateText).toHaveAttribute('min', '01/01/2020');
   });
+
+  /*
```

Comment:
Should we just .skip this?

Reply:
I really do feel like we should complete this test.  It's bugging me.  I know it should be simple, and I'm not sure why it's not.

If we punt on it, then yes we should skip it and leave a comment as to why.